backend/app/Actions/Teams/RemoveTeamMemberAction.php

```diff
@@ -23,13 +21,11 @@ public function __construct(
         $this->websiteRepository = $websiteRepository;
     }
 
-    public function execute(RemoveTeamMemberRequest $request): void
+    public function execute($userId): void
```

Comment:
Jesus, how many times we should point it in order to you remember this? Type hinting is required, a parameter cannot be without the type.